proto/xudrpc.proto

```diff
@@ -162,14 +160,12 @@ message GetOrdersRequest {
   uint32 max_results = 2 [json_name = "max_results"];
 }
 message GetOrdersResponse {
```

Comment:
Can you also update the description of the `GetOrders` call (line 62) to a better description of what it does, I think it's quite out of date.